### PR TITLE
refactor: 🚚 adjust module naming to support go install

### PR DESCRIPTION
**Pull request by sheldonhull**
- Add golangci-lint configuration for local linting checks.
- Add DS_Store to .gitignore
- Include linting tasks in Makefile.

These modifications should support more easily using in CI tooling and linting by support for remote install with `go install github.com/bvieira/sv4git/sv@latest`.

### CODE REVIEW

**Comment by sheldonhull** [comment]
@bvieira  doing a direction mention to make sure this emailed you. Cheers

File: Makefile [hunk 1]
```diff
@@ -32,6 +32,16 @@ build: test
 	@echo $(ECHOFLAGS) "$(OK_COLOR)==> Building binary ($(BUILDOS)/$(BUILDARCH)/$(BIN))...$(NO_COLOR)"
 	@$(BUILDENVS) go build -v $(BUILDFLAGS) -o bin/$(BUILDOS)_$(BUILDARCH)/$(BIN) ./cmd/git-sv
 
+## list: run golangci-lint without autofix
```
Comment (bvieira) [inline Makefile:35]:
```suggestion
## lint: run golangci-lint without autofix
```

File: Makefile [hunk 2]
```diff
@@ -32,6 +32,16 @@ build: test
...
+lint:
+	@echo $(ECHOFLAGS) "$(OK_COLOR)==> Running golangci-lint...$(NO_COLOR)"
+	@golangci-lint run ./... --config .golangci.yml
+
+## autofix: run golangci-lint with autofix enabled
```
Comment (bvieira) [inline Makefile:40]:
```suggestion
## lint-autofix: run golangci-lint with autofix enabled
```

File: Makefile [hunk 3]
```diff
@@ -32,6 +32,16 @@ build: test
...
+autofix:
```
Comment (bvieira) [inline Makefile:41]:
```suggestion
lint-autofix:
```

**Comment by bvieira** [comment]
hi @sheldonhull,

Thanks for your pull request :clap:  

I've been thinking about adding a linter, thanks for starting for me! There some configs that should be changed at `.golangci.yml`, but I can do it after merging.

If you could just accept my suggestions o makefile, I would be glad to merge your PR :wink:

**Review by bvieira — APPROVED** [review]

**Comment by sheldonhull** [comment]
@bvieira  thank you for the fixes. Didn't get a chance to look at this till now. Woot woot. 🚀

**Comment by sheldonhull** [comment]
@bvieira  suggest adding this to the readme for easier install using Go command.

```shell
go install github.com\bvieira\sv4git\cmd\git-sv@master
```

Latest release (once you have a new tagged release. Currently this won't work as it's the older version)

```shell
go install github.com\bvieira\sv4git\cmd\git-sv@latest
```

**Pull request merged**